.github/workflows/deploy.yml

```diff
@@ -98,7 +98,7 @@ jobs:
     - name: create tags
       id: tags
       run: |
-        IMG=quay.io/${{ github.repository_owner }}/${{ matrix.id }}
+        IMG=quay.io/${{ secrets.QUAY_USERNAME  }}/${{ matrix.id }}
```

Comment:
TODO switch this back. Using a secret here probably causes  ******** in the logs.